src/strace.c

```diff
@@ -2965,18 +2996,21 @@ init(int argc, char *argv[])
 
 	/* See if they want to run as another user. */
 	if (username != NULL) {
-		struct passwd *pent;
-
 		if (getuid() != 0 || geteuid() != 0) {
 			error_msg_and_die("You must be root to use "
 					  "the -u/--username option");
 		}
-		pent = getpwnam(username);
-		if (pent == NULL) {
-			error_msg_and_die("Cannot find user '%s'", username);
+
+		// If the username is in the form UID:GID then do not perform lookup.
```

Comment:
strace prefers /* */ comments.

Reply:
Sorry, old habits die hard. I din't notice re-introducing this.